FitGoal/FitGoal/Setting/SettingsViewController.swift

```diff
@@ -20,5 +28,58 @@ class SettingsViewController: UITableViewController {
             userFitnessLevel: "Intermediate",
             userProfileImage: nil
         )
+        self.tableView.tableFooterView = UIView()
+        self.tableView.register(UserInfoCell.self, forCellReuseIdentifier: UserInfoCell.identifier)
+    }
+    
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return cellInformation.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: UserInfoCell.identifier, for: indexPath)
+        guard let userInfoCell = cell as? UserInfoCell else { fatalError() }
+        userInfoCell.selectionStyle = .none
+        userInfoCell.cellInformation = cellInformation[indexPath.row]
+        return userInfoCell
+    }
+    
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let header = UIView(frame: .zero)
+        let sectionName = UILabel()
+        sectionName.attributedText = "user info".uppercased().formattedText(
+            font: "Roboto-Bold",
+            size: 15,
+            color: #colorLiteral(red: 0.501960814, green: 0.501960814, blue: 0.501960814, alpha: 1),
+            kern: 0.14
+        )
+        header.addSubview(sectionName)
+        sectionName.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            sectionName.leadingAnchor.constraint(equalTo: header.leadingAnchor, constant: 16),
+            sectionName.topAnchor.constraint(equalTo: header.topAnchor, constant: 24)
+        ])
+        return header
+    }
+    
+    override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 50
+    }
+    
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 50
     }
 }
+
+struct CellInformation {
```

Comment:
This type is quite generic. What about using a richer name that gives a clue about what that type is for?